File: Visualization_Msc_Sem03/Exercise2/FarbRechner/FarbRechner/VBO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FarbRechner
{
    /// <summary>
    /// Container class for our VertexBufferObjects, for better handling and to possibly avoid ambiguitations
    /// </summary>
    /// <author>Markus Strobel</author>
    public class VBO
    {
        public int VertexBufferID { get; set;}
        public int IndicesBufferID { get; set; }
        public int NumElements { get; set; } 

        public VBO()
        {
            VertexBufferID = -1;
            IndicesBufferID = -1;
            NumElements = -1;
        }
    }
}

File: Visualization_Msc_Sem03/Exercise3/src/bigdata/VBO.java
package bigdata;

/**
 * Vertex Buffer Object Class
 * @author Markus Strobel
 *
 */
public class VBO {

    public int VertexBufferID;
    public int IndicesBufferID;
    public int NumElements;
    public int NumIndices;
    
    public VBO()
    {
    	this(-1, -1, -1, -1);
    }
    
    public VBO(int verticesBufferId, int indicesBufferId, int numElements, int numIndices)
    {
    	VertexBufferID = verticesBufferId;
    	IndicesBufferID = indicesBufferId;
    	NumElements = numElements;
    	NumIndices = numIndices;
    }
}
